app/index.html

```diff
@@ -1,12 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <!-- Google Tag Manager -->
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-5S2P6SD');</script>
+<!-- End Google Tag Manager -->
```

Comment:
Even though this is just a static file that's separate from the rest of the app, can we indent this to match the indentation of the rest of the file? Children of an element should be indented one level (two spaces) inward, and all sibling elements should be at the same indentation level.

Also, since this isn't a human readable snippet anyway, can we just put this entire `<script>` tag on one line? Putting line breaks arbitrarily like this makes it slightly more difficult to scan the file, since it's hard to see when the snippet starts and when the snippet ends.

Reply:
Also, just to be absolutely clear, even though HTML comments (`<!-- this stuff -->`) aren't real elements, I would appreciate if they also follow the same indentation rules.